package.json

```diff
@@ -57,11 +57,9 @@
     "endOfLine": "lf"
   },
   "devDependencies": {
-    "@types/brace-expansion": "^1.1.0",
```

Comment:
This is necessary for typescript compilation.

Reply:
I didn't get any warnings/errors from `tsc` as this package is only a subdep of tap, but I added it back anyway

Reply:
Oh! lol, I thought this was minimatch, sorry I guess I have too many modules 😅 Yes, that can be removed. I'll get this merged, don't worry about updating a second time.